GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2677,7 +2677,7 @@ private void createBranchToolStripButton_Click(object sender, EventArgs e)
 
         private void Message_Enter(object sender, EventArgs e)
         {
-            if(Staged.AllItems.Count() != 0 && Staged.SelectedItems.Count() == 0)
+            if(Staged.AllItems.Count() != 0 && !Staged.SelectedItems.Any())
```

Comment:
Sorry for including this. Either ReSharper or VS suggested this change which is a good one. In the Count() case have to enumerate all the items even though we really care to know if there is at least one.